app/controllers/sign_up/registrations_controller.rb

```diff
@@ -64,5 +68,20 @@ def redirect_if_ial2_and_idv_unavailable
         redirect_to idv_unavailable_path(from: CREATE_ACCOUNT)
       end
     end
+
+    def threat_metrix_variables
+      return unless IdentityConfig.store.account_creation_device_profiling_enabled
+      session_id = generate_threatmetrix_session_id
+
+      {
+        threatmetrix_session_id: session_id,
+        threatmetrix_javascript_urls: session_id && threatmetrix_javascript_urls(session_id),
+        threatmetrix_iframe_url: session_id && threatmetrix_iframe_url(session_id),
+      }
```

Comment:
it seems like if we get here, `session_id` will always be present? so we can remove the conditionals?
```suggestion
      session_id = generate_threatmetrix_session_id

      {
        threatmetrix_session_id: session_id,
        threatmetrix_javascript_urls: threatmetrix_javascript_urls(session_id),
        threatmetrix_iframe_url: threatmetrix_iframe_url(session_id),
      }
```